tests/src/shader-compiler/Precompile.test.ts

```
@@ -1077,6 +1077,41 @@ describe("ShaderCompiler Precompile", async () => {
+        ["V_FN", "mix(u_fn_a, u_fn_b, 0.5)"],
+        ["V_UNARY", "-u_unary"],
+        ["V_SKY", "(mix(0.0, 0.0025, pow(material_AtmosphereThickness, 2.5)))"]
+      ];
+      for (const [name, value] of expectedDefines) {
+        const inst = defines.get(name);
+        expect(inst, `missing #define ${name}`).toBeDefined();
+        expect(inst![0], `${name} must be DefineVal (8), got ${inst![0]}`).toBe(8);
+        expect(inst![2]).toBe(value);
+      }
+
+      const fragText = fragInstr.filter((i) => i[0] === 0).map((i) => i[1] as string).join("");
```

ESLint

[error] Replace `.filter((i)·=>·i[0]·===·0).map((i)·=>·i[1]·as·string)` with `⏎········.filter((i)·=>·i[0]·===·0)⏎········.map((i)·=>·i[1]·as·string)⏎········`

(prettier/prettier)